src/utils/wallet.ts

```diff
@@ -332,7 +333,25 @@ const wallet = {
 
     const derivedXpub = xpub.deriveChild(derivationIndex);
     return derivedXpub.xpubkey;
-  }
+  },
+
+  /**
+ * Calculate the double sha256 hash of the data.
+ *
+ * @remarks
+ * If encoding is provided a string will be returned; otherwise a Buffer is returned.
+ *
+ * @param data - Data to be hashed
+ * @param encoding - The encoding of the returned object
+ * @returns The sha256d hash of the data
+ */
+  sha256d(data: string, encoding: HexBase64Latin1Encoding): string {
```

Comment:
```suggestion
  sha256d(data: string, encoding?: HexBase64Latin1Encoding): string | Buffer {
```

Reply:
Done in c8600b30a1fb0f5b8da60993a30dd65205e23505